src/main/java/org/lobid/resources/ElasticsearchIndexer.java

```diff
@@ -133,8 +133,13 @@ public void onCloseStream() {
 				&& !aliasSuffix.toLowerCase().contains("test"))
 			updateAliases();
 		// feed the rest of the bulk
-		if (bulkRequest.numberOfActions() != 0)
-			bulkRequest.execute().actionGet();
+		if (bulkRequest.numberOfActions() != 0) {
+			System.out.println(bulkRequest.toString());
```

Comment:
This looks like it was meant to be temporary.

Reply:
good catch!